docs/UserGuide.md

```diff
@@ -1,186 +1,67 @@
-# AddressBook Level 4 - User Guide
+# User Guide
 
-By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbsp;&nbsp; Licence: `MIT`
+This product is a tool to accept natural language commands via keyboards to manage the user's schedule and todo tasks.
 
----
+## Start the program
```

Comment:
You are going to provided `.jar` file in the end. We run your application through `.jar` file rather than use eclipse.